bin/build-helper

```diff
@@ -17,9 +17,21 @@ createImagesFileIfNotExists() {
   fi
 }
 
+setRegistry() {
```

Comment:
Maybe rename to `formatRegistry`, since the registry value's actually set in the env?

Reply:
This has been updated.